manim/__main__.py

```diff
@@ -77,10 +72,10 @@ def main():
     else:
         config.digest_args(args)
         input_file = config.get_dir("input_file")
-        if config["use_js_renderer"]:
+        if config["use_webgl_renderer"]:
             try:
-                if frame_server_impl is None:
-                    raise ImportError("Dependencies for JS renderer is not installed.")
+                from manim.grpc.impl import frame_server_impl
+
```

Comment:
I would like to have this error caught and the user informed that there are missing dependency and also a way to install then.

Reply:
The ModuleNotFoundError is already printed; there's no reason to rephrase what it's already saying.

Reply:
But it doesn't say how to install it though.  I think this should be done if we are doing something as optional. We should inform users on Import error that run
```sh
pip install manim[webgl_renderer]
```
to install the dependencies needed to run webgl renderer.